src/bus/event_bus.cpp

```diff
@@ -0,0 +1,9 @@
+#include "bus/event_bus.hpp"
+
+namespace mesos {
+namespace serenity {
+
+EventBus* EventBus::instance = nullptr;
```

Comment:
Not necessarily singleton - we could have multiple instances however we need static pointer to them, since we would like to have access to that from different modules within same lib.